app/controllers/users/invitations_controller.rb

```diff
@@ -53,12 +53,8 @@ def load_and_authorize_role
       when ClientSuccessRole::TYPE
         ClientSuccessRole.new
       when GreeterRole::TYPE
-        greeter_params = params.require(:greeter_organization_join_record).permit(organization_ids: []).merge(
-          params.require(:greeter_coalition_join_record).permit(coalition_ids: [])
-        )
         GreeterRole.new(
-          coalitions: @coalitions.where(id: greeter_params[:coalition_ids]),
-          organizations: @vita_partners.organizations.where(id: greeter_params[:organization_ids]),
+          organizations: @vita_partners.organizations.where(allows_greeters: true),
```

Comment:
This is going to assign the greeter role to all users who have allows_greeters set at that moment in time. But when this list changes, the list of organizations this greeter has access to _wont_ change. Instead, I think we want the greeter role to act like the admin role -- we won't save any relationships to it at all right now. Instead, we should restrict the clients the user has access to through the `accessible_organizations` method on User model.